CI/PHP-CS-Fixer/code-format.php_cs

```diff
@@ -26,5 +26,8 @@ return PhpCsFixer\Config::create()
         'unary_operator_spaces' => true,
         'function_typehint_space' => true,
         'return_type_declaration' => ['space_before' => 'one'],
+        'binary_operator_spaces' => [
+         'operators' => ['=' => 'align_single_space']
```

Comment:
```suggestion
         'operators' => ['=' => 'single_space']
```

Reply:
@klees thx for the review, the config line is changed.